snuba/request/schema.py

```diff
@@ -104,6 +108,18 @@ def validate(self, value, dataset: Dataset, referrer: str) -> Request:
             }
 
         query = parse_query(query_body, dataset)
+        # Temporary code to collect some metrics about the usage of
```

Comment:
The place for this code should be in the parser (since there is where aliases resolution and management happens), though the parser does not know the referrer. Anyway I am going to collect at most a day worth of data and then revert